modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/recurrence.js

```diff
@@ -43,6 +44,7 @@ AUI.add(
 				},
 
 				dayOfWeekInput: {
+					// eslint-disable-next-line @liferay/aui/no-one
 					setter: A.one,
```

Comment:
these properties being automatically converted to AUI format make me 😱 .

For some properties, like `dayOfWeekInput` a `.val()` is being used. Are you planning to remove these ones?

Reply:
Oh, I saw https://github.com/liferay-frontend/liferay-portal/pull/1497/files#diff-653649ff35bd46e7cdeb9d7891ee2e267227b6c9eb9f3eefde427e67350a1857R517. You are migrating xD

Reply:
![](https://media.giphy.com/media/1jkVi22T6iUrQJUNqk/giphy.gif?cid=ecf05e479lztbi6mhml9dk7nbhcux5jrfdaiy1gdxrmpmyi4&rid=giphy.gif&ct=g)

Reply:
Yea, I think the plan is to replace all the straight forward ones, and add the ignore comment for those that rely on AUI versions in their implementation, like that `dayOfWeekInput` there. If we wanted to change that to a vanilla selector, then who know what would break because the implementation of that class is expecting an AUI node.